client/sites-dashboard/controller.tsx

```diff
@@ -129,6 +131,19 @@ function sitesDashboard( context: PageJSContext, next: () => void ) {
 		}
 	`;
 
+	// Update body margin to account for the sidebar width if the new nav is enabled
+	if ( isEnabled( 'layout/dotcom-nav-redesign' ) ) {
```

Comment:
Not using hook here because I couldn't figure out a way to do this from the controller. I think this method works ok for now because this is the site dashboard controller (so the conditions in `useGlobalSidebar` would be met, i.e. the check that the sectionGroup is `sites-controller`.